lighthouse-core/config/default-config.js

```diff
@@ -416,17 +416,41 @@ const defaultConfig = {
       title: str_(UIStrings.bestPracticesGeneralGroupTitle),
     },
   },
+};
+
+// go/lh-audit-metric-mapping
+const fcpRelevantAudits = [
+  // Roughly in order of impact
```

Comment:
is order used at all? What is impact? If none of this is used or well defined, I'd prefer just alphabetical order (and one per line :)